tests/performance/pod-oauth.yaml

```diff
@@ -59,14 +63,16 @@ spec:
           value: "1000000"
         - name: TS_SELENIUM_WORKSPACE_STATUS_POLLING
           value: "500000"
+        - name: TS_LOAD_TESTS
```

Comment:
What is the benefit of adding one more constant to support if we continue using TEST_SUITE = "load-test" ?